internal/usage/usage.go

```diff
@@ -30,6 +30,7 @@ const (
 	DiskSizeGB               = "Capacity, in gigabytes, of the host's root volume."
 	Backup                   = "If true, enables Continuous Cloud Backup for your deployment. This option is unavailable for clusters smaller than M10."
 	BIConnector              = "If true, enables BI Connector for Atlas on the deployment."
+	TerminationProtect       = "If true, enable termination protection for your cluster."
```

Comment:
Adding a little context on what termination protection does.

```suggestion
	TerminationProtect       = "If true, enable termination protection for your cluster. You can't delete the cluster with termination protection enabled. "
```